docs/contribute/module.md

```diff
@@ -149,3 +149,11 @@ changes. Include:
 - a link to your release notes. 
 
 Request a review from someone in the core team and wait for their feedback!
+
+## Adding your module to the autora parent package
```

Comment:
You also should include the docs and package in the mkdocs.yml and pyproject.toml file of autora parent and create a pull request, right? Or do members of the tem do that?